syncd/syncd_hard_reinit.cpp

```diff
@@ -1098,6 +1099,45 @@ void processRoutes(bool defaultOnly)
     }
 }
 
+void processNatEntries()
```

Comment:
@kcudnik , can you the code logic here?

Reply:
I mean this function processNatEntries.

Reply:
i dont understand the question "can you the code logic here?" what this mean ? do you want to to review the code (obvious since i review entire PR) can you explain more ?